packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -120,7 +120,7 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
         maxWidth: "374px",
         minWidth: ensureDesignSystemDefaults(
             (designSystem: DesignSystem): string =>
-                designSystem.density <= -2 ? "100px" : "120px"
+                designSystem.density <= -2 ? "28px" : "32px"

```

Comment:
should this be a function of the design unit?

Reply:
It isn't anywhere else, so I'm a bit hesitant to swap the logic out for something more dynamic right now. Considering this pattern exists throughout the code, I'd prefer we make a change like that in a new PR after consulting with @bheston who drove this as part of the density changes.